blaze-client/src/test/scala/org/http4s/client/blaze/BlazeClientSpec.scala

```diff
@@ -113,41 +111,28 @@ class BlazeClientSpec extends Http4sSpec {
             .forall(_.contains(true)) must beTrue
         }
 
-        "obey response line timeout" in {
+        "obey response header timeout" in {
```

Comment:
I can't make much sense of what the previous test did, and it's flaky.

This is somewhat redundant with the ClientTimeoutSpec.